CRM/Core/BAO/UFField.php

```diff
@@ -221,24 +219,6 @@ public static function add(&$params, $ids = array()) {
     }
 
     $ufField->phone_type_id = CRM_Utils_Array::value(3, $params['field_name'], 'NULL');
-    $ufField->listings_title = CRM_Utils_Array::value('listings_title', $params);
-    $ufField->visibility = CRM_Utils_Array::value('visibility', $params);
-    $ufField->help_pre = CRM_Utils_Array::value('help_pre', $params);
-    $ufField->help_post = CRM_Utils_Array::value('help_post', $params);
-    $ufField->label = CRM_Utils_Array::value('label', $params);
-    $ufField->is_required = CRM_Utils_Array::value('is_required', $params, FALSE);
```

Comment:
0 is the DB default for is_required, in_selector, is_view, is_searchable,  is_multi_summary,

1 is the DB default for is_active - which seems more correct that the default applied here!

is_registration & is_match don't exist!

Which leaves is_reserved unchanged - but which really should have a db default of 0.

Note the behaviour WILL change for update - because unchanged fields won't be altered unless that is specifically desirable,